shell/platform/tizen/BUILD.gn

```diff
@@ -200,18 +195,25 @@ template("embedder") {
         "elementary",
         "evas",
       ]
-
-      public_configs += [ ":evas_gl_renderer" ]
     } else {
       sources += [
+        "external_texture_pixel_egl.cc",
+        "external_texture_surface_egl.cc",
         "flutter_tizen_ecore.cc",
+        "flutter_tizen_elementary.cc",
         "tizen_renderer_egl.cc",
+        "tizen_renderer_evas_gl.cc",
+        "tizen_view_elementary.cc",
         "tizen_vsync_waiter.cc",
         "tizen_window_ecore_wl2.cc",
+        "tizen_window_elementary.cc",
       ]
 
       libs += [
+        "ecore_evas",
         "ecore_wl2",
+        "elementary",
+        "evas",
```

Comment:
You can avoid the duplication by adding sources and libs that are common to all profiles to `sources` and `libs` at line 103 and 139.